akka-docs-dev/rst/java/migration-guide-1.0-2.x-java.rst

```diff
@@ -455,4 +455,87 @@ Example
 
 should be replaced by
 
-.. includecode:: code/docs/MigrationsJava.java#file-source-sink
\ No newline at end of file
+.. includecode:: code/docs/MigrationsJava.java#file-source-sink
+
+InputStreamSource and OutputStreamSink
+======================================
+
+Both have been replaced by `Source.inputStream(…)` and `Sink.outputStream(…)` due to discoverability issues.
+
+Update procedure
+----------------
+
+Replace `InputStreamSource.create(` with `Source.inputStream(`
+
+Replace `OutputStreamSink.create(` with `Sink.outputStream(`
```

Comment:
rst syntax is: 

```
Replace ``OutputStreamSink.create(`` with ``Sink.outputStream(``
```

Yep, markdown would be simpler, but we're not changing at the moment..